app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
 
   supports :provisioning
   supports :refresh_new_target
+  supports :vm_import do
+    # The version of the RHV needs to be at least 4.1.5 due to https://bugzilla.redhat.com/1477375
+    version_higher_than?('4.1.5')
```

Comment:
note that this method was recently renamed https://github.com/ManageIQ/manageiq-providers-ovirt/pull/143#pullrequestreview-79021137

Reply:
Done.